tests/test_ingress.py

```diff
@@ -166,6 +167,54 @@ def test_redo_failed_messages_fails_with_non_existing_endpoint(self):
+        self.assertEqual(Endpoint.objects.count(), 2)
+
+        # Add a couple failed messages for both the endpoints
+        first_endpoint = Endpoint.objects.get(url_key='first_endpoint')
+        second_endpoint = Endpoint.objects.get(url_key='second_endpoint')
+        for _ in range(3):
+            one = FailedIngressQueue.objects.create(
+                endpoint=first_endpoint,
+                parse_started='2020-01-12 12:12:12',
+                raw_data='the data in first endpoint')
+            one.created_at = '2020-01-12 12:12:12'  # Overwrite the default created_at to make it testable
+            one.save()
+            two = FailedIngressQueue.objects.create(
+                endpoint=second_endpoint,
+                parse_started='2020-01-12 12:12:12',
+                raw_data='the data in second endpoint'
+            )
+            two.created_at = '2020-01-12 12:12:12'  # Overwrite the default created_at to make it testable
+            two.save()
+        self.assertEqual(IngressQueue.objects.count(), 0)
+        self.assertEqual(FailedIngressQueue.objects.count(), 6)
+
+        # First select one failed message without selecting an endpoint
+        out = call_man_command('show_failed_message')
+        expected_output = '\n\nendpoint             first_endpoint                                    \n' \
+                          'created_at           2020-01-12 11:12:12+00:00                         \n' \
+                          'parse_started        2020-01-12 11:12:12+00:00                         \n' \
+                          'parse_succeeded      None                                              \n' \
+                          'parse_failed         None                                              \n' \
+                          'parse_fail_info      None                                              \n' \
+                          'raw_data             the data in first endpoint                        \n'
+        self.assertEqual(out, expected_output)
+
+        # Then select one failed message from the second endpoint
+        out = call_man_command('show_failed_message', 'second_endpoint')
+        expected_output = '\n\nendpoint             second_endpoint                                   \n' \
+                          'created_at           2020-01-12 11:12:12+00:00                         \n' \
+                          'parse_started        2020-01-12 11:12:12+00:00                         \n' \
+                          'parse_succeeded      None                                              \n' \
+                          'parse_failed         None                                              \n' \
+                          'parse_fail_info      None                                              \n' \
+                          'raw_data             the data in second endpoint                       \n'
+        self.assertEqual(out, expected_output)
```

Comment:
Dit kan ook korter:
```
out = call_man_command('show_failed_message')
expected_output = get_expected_output(first_failed)
self.assertEqual(out, expected_output)

out = call_man_command('show_failed_message', 'second_endpoint')
expected_output = get_expected_output(first_failed2)
self.assertEqual(out, expected_output)
```

Reply:
`get_expected_output` kan dan op basis van de input de output str maken.

Reply:
Eens, maar een test is niet alleen om te testen of het werkt, maar ook als documentatie. En persoonlijk zou ik het heerlijk vinden om dit te zien in tests, zodat ik meteen zie wat het produceert. Ook omdat deze output echt bedoeld is om door een gebruiker te lezen.